test/Dialect/LLHD/Transforms/desequentialization.mlir

```diff
@@ -135,3 +135,27 @@ hw.module @asyncResetNotObserved(inout %rst : i1, inout %clk : i1, inout %sig :
     cf.br ^bb1
   }
 }
+
+// CHECK-LABEL: @compareClkPrb
+// CHECK-SAME: (inout [[CLK:%.+]] : i1, inout [[OUT:%.+]] : i1)
+hw.module @compareClkPrb(inout %clk : i1, inout %out : i1) {
+  // CHECK: [[V0:%.+]] = llhd.prb [[CLK]]
+  // CHECK: [[V1:%.+]] = seq.to_clock [[V0]]
+  // CHECK: [[V2:%.+]] = comb.xor [[V0]], %true{{.*}}
+  // CHECK: [[V3:%.+]] = seq.compreg %false{{.*}}, [[V1]] reset [[V2]], %false{{.*}}
```

Comment:
Note: this is currently incorrect as it should be an async reset, but `seq.compreg` does not support that (we need a register in the seq dialect that supports that without having the FIRRTL specific parts of firreg).
No need to change anything in this PR.